builder/evaluator.go

```diff
@@ -74,7 +74,7 @@ func init() {
 	}
 }
 
-// internal struct, used to maintain configuration of the Dockerfile's
+// Builder is an internal struct, used to maintain configuration of the Dockerfile's
```

Comment:
Funny, we have an internal struct that is part of the public API of this package.